### ISSUE TITLE
Determine method for adding new build task types

### ISSUE THREAD

**Issue by drewbo** [cloud-gov/pages-core#4260]
Once the first task type is complete, consider how new tasks will be added. Two potential options are via the admin UI or via database migrations

## Acceptance Criteria

- [ ] Determine next steps and acceptance criteria

**Comment by drewbo** [comment]
To ensure consistency between environments, it will be easiest to add new task types via migration. A rough outline of adding a new type:
- add a new CF task in each environment (ideally via concourse)
- add a migration which takes `env` as an environment variable, uses it to add the required metadata for the new build task

**Comment by drewbo** [comment]
After the task type exists, we should separately consider how to add a new `site_build_task`
